mousestyles/dynamics/__init__.py

```diff
@@ -263,3 +263,74 @@ def get_prob_matrix_small_interval(string_list):
         if sum(Mat_prob[k, :]) != 0:
             Mat_prob[k, :] = Mat_prob[k, :]/sum(Mat_prob[k, :])
     return Mat_prob
+
+
+def mcmc_simulation(mat_list, n_per_int):
+    r"""
+    This function gives the Monte Carlo simulation
+    of the stochastic process modeling the dynamic
+    changes of states of the mice behavior. The in-
+    put of this function is a list of probability
+    transition matrices and an integer indicates
+    how many outputs for each matrix. This number
+    is related to the interval_length parameter in
+    function get_prob_matrix_list. The output is an
+    array of numbers, each indicate one state.
+
+    Parameters
+    ----------
+    mat_list: List
+        a list of numpy arrays storing the probabi-
+        lity transition matrices for each small time
+        interval chosen.
+    n_per_int: int or float
```

Comment:
I will comment that this n_per_int and the interval_length are essentially the same thing in the doc string.